### DISCUSSION TITLE
Announcement bar X button still shows even if `isCloseable` is set to `false`.

### DISCUSSION THREAD

**Discussion by GodderE2D** [facebook/docusaurus#5325] [Q&A] [answered]
In my `docusaurus.config.js`:
```js
announcementBar: {
  id: 'announcementbar',
  content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris faucibus.',
  backgroundColor: '#25c2a0'.
  textColor: '#2c2f33',
  isCloseable: false,
}
```
When I visit the site, the announcement bar shows up as expected, however, the X button still appears and when I click it, it disappears, which is not what I want.

### Expected
The X button should not be showing and non-functional.
### Result
The X button shows, and the announcement bar disappears when clicked.

I am opening a discussion because I have seen other websites with announcement bars that are non-collapsible, but for some reason, this method does not work.

### Version
`@docusaurus/core`: `2.0.0-alpha.54`
`@docusaurus/preset-classic`: `2.0.0-alpha.54`
`classnames`: `2.2.6`
`react`: `16.8.4`
`react-dom`: `16.8.4`
Node: `16.6.1`
Yarn: `1.22.11`

**Comment by slorber** [comment, marked as answer]
It's important to upgrade your Docusaurus site before reporting bugs. We don't provide support for older versions.
